### ISSUE TITLE
Logging does not work on local dev docker

### ISSUE THREAD

**Issue by jonfroehlich** [makeabilitylab/makeabilitylabwebsite#984]
This has been a long-standing issue (in fact, just found a duplicate here https://github.com/makeabilitylab/makeabilitylabwebsite/issues/815). Perhaps I can discuss with Jason. Appears to be another one of those pesky permissions issues:

```
website_1  | Traceback (most recent call last):
website_1  |   File "/usr/local/lib/python3.8/logging/config.py", line 563, in configure
website_1  |     handler = self.configure_handler(handlers[name])
website_1  |   File "/usr/local/lib/python3.8/logging/config.py", line 744, in configure_handler
website_1  |     result = factory(**kwargs)
website_1  |   File "/usr/local/lib/python3.8/logging/handlers.py", line 148, in __init__
website_1  |     BaseRotatingHandler.__init__(self, filename, mode, encoding, delay)
website_1  |   File "/usr/local/lib/python3.8/logging/handlers.py", line 55, in __init__
website_1  |     logging.FileHandler.__init__(self, filename, mode, encoding, delay)
website_1  |   File "/usr/local/lib/python3.8/logging/__init__.py", line 1147, in __init__
website_1  |     StreamHandler.__init__(self, self._open())
website_1  |   File "/usr/local/lib/python3.8/logging/__init__.py", line 1176, in _open
website_1  |     return open(self.baseFilename, self.mode, encoding=self.encoding)
website_1  | PermissionError: [Errno 13] Permission denied: '/code/media/debug.log'
website_1  |
website_1  | The above exception was the direct cause of the following exception:
website_1  |
website_1  | Traceback (most recent call last):
website_1  |   File "manage.py", line 10, in <module>
website_1  |     execute_from_command_line(sys.argv)
website_1  |   File "/usr/local/lib/python3.8/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
website_1  |     utility.execute()
website_1  |   File "/usr/local/lib/python3.8/site-packages/django/core/management/__init__.py", line 357, in execute
website_1  |     django.setup()
website_1  |   File "/usr/local/lib/python3.8/site-packages/django/__init__.py", line 19, in setup
website_1  |     configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
website_1  |   File "/usr/local/lib/python3.8/site-packages/django/utils/log.py", line 76, in configure_logging
website_1  |     logging_config_func(logging_settings)
website_1  |   File "/usr/local/lib/python3.8/logging/config.py", line 808, in dictConfig
website_1  |     dictConfigClass(config).configure()
website_1  |   File "/usr/local/lib/python3.8/logging/config.py", line 570, in configure
website_1  |     raise ValueError('Unable to configure handler '
website_1  | ValueError: Unable to configure handler 'file'
website_1  | Traceback (most recent call last):
website_1  |   File "/usr/local/lib/python3.8/logging/config.py", line 563, in configure
website_1  |     handler = self.configure_handler(handlers[name])
website_1  |   File "/usr/local/lib/python3.8/logging/config.py", line 744, in configure_handler
website_1  |     result = factory(**kwargs)
website_1  |   File "/usr/local/lib/python3.8/logging/handlers.py", line 148, in __init__
website_1  |     BaseRotatingHandler.__init__(self, filename, mode, encoding, delay)
website_1  |   File "/usr/local/lib/python3.8/logging/handlers.py", line 55, in __init__
website_1  |     logging.FileHandler.__init__(self, filename, mode, encoding, delay)
website_1  |   File "/usr/local/lib/python3.8/logging/__init__.py", line 1147, in __init__
website_1  |     StreamHandler.__init__(self, self._open())
website_1  |   File "/usr/local/lib/python3.8/logging/__init__.py", line 1176, in _open
website_1  |     return open(self.baseFilename, self.mode, encoding=self.encoding)
website_1  | PermissionError: [Errno 13] Permission denied: '/code/media/debug.log'
```

**Comment by jonfroehlich** [comment]
Interestingly, when I went in and fixed https://github.com/makeabilitylab/makeabilitylabwebsite/issues/985, it somehow also fixed this issue. So, clearly a permissions issue of some sort.

The key was to make a bash shell inside the container and make sure /code/media had correct permissions.

**Comment by jonfroehlich** [comment]
OK, so I figured out how to solve it on Windows WSL2 with ubuntu. The problem is a permissions issue with `/code/media` in the Docker container.

By default, it looks like `/code/media` has a permission of `drwxr-xr-x  2 root root  4096 Oct 13 02:59 media`. I just changed this to 777 and it worked. Here are the steps:

1. Open the Docker container in a bash shell: `> docker exec -it makeabilitylabwebsite_website_1 bash`
2. Make sure you are in the `/code` directory. Try to change the permissions by `$ chmod 777 media`
3. If that doesn't work, you'll need sudo access. So, try: 
    ```
    apache@e86363656952:/code$ su
    Password:
    root@e86363656952:/code# chmod 777 media
    ```
4. Restart the Docker container (e.g., from Ubuntu, `> docker compose-up`
